cmd/src/batch_common.go

```diff
@@ -176,14 +194,31 @@ func batchDefaultTempDirPrefix() string {
 	return os.TempDir()
 }
 
-func batchOpenFileFlag(flag *string) (io.ReadCloser, error) {
-	if flag == nil || *flag == "" || *flag == "-" {
+func batchOpenFileFlag(flag string) (io.ReadCloser, error) {
```

Comment:
this is also used in the exec path - not sure we want this there? anyways, not super important.

Reply:
It's not particularly relevant, but I also don't think it'll cause problems. We can pull out the open/validate bits we need for `exec` at a later date if we want. (I don't mind doing that, but I also don't think it's on the critical path here.)

Reply:
yeah no, let's leave it as is.